libs/wingsdk/src/target-sim/service.inflight.ts

```diff
@@ -55,7 +55,7 @@ export class Service implements IServiceClient, ISimulatorResourceInstance {
       sourceType: SERVICE_TYPE,
       timestamp: new Date().toISOString(),
     });
-    await fnClient.invoke("");
+    fnClient.invoke("");
```

Comment:
await a second 😁 , we need to ignore the `@typescript-eslint/no-floating-promises` here I think

Reply:
Thanks!